src/providers/apolloErrorLink.tsx

```diff
@@ -52,13 +52,27 @@ const apolloErrorLink = onError(
       });
 
       if (networkError) {
+        // check if error message is probably an http status code
+        const statusCode = networkError.message.match(/^[4-5][0-9][0-9]$/)
+          ? parseInt(networkError.message)
+          : undefined;
+        if (statusCode) scope.setTag('status_code', statusCode);
+
         // Not a server error. This is usually 504 or "Network request failed", track in Sentry
         if (!isServerError(networkError)) {
-          if (recordedErrors === 0) Sentry.captureException(networkError);
```

Comment:
it doesn't seem like we get both graphql and network errors in this case. Skip this check